src/Cryptogram.java

```diff
@@ -0,0 +1,35 @@
+import java.io.*;
+import java.util.*;
+
+public class Cryptogram{
+    
+    public List<String> cryptogram;
+    public HashMap<Character, Integer> frequencyCount;
+
+    public Cryptogram(List<String> words) throws FileNotFoundException{
+
+        this.cryptogram = words;
+        
+        frequencyCount = new HashMap<Character, Integer>();
+        for(int i = 0; i < words.size(); i++){
```

Comment:
Thankfully, we are using a version of java where we can do a fast enumeration.  This for-loop would then look like: `for (String word: words) {`, and `words.get(i)` would become `word`.  Much easier to read IMO.